### PR TITLE
infra: update templates to new form style, add config links

### PR DESCRIPTION
**Pull request by JacobCoffee**
#### Description

- updates to new issue templates and adds some quick links to help steer lost people in the right direction

<details>
<summary>Details</summary>

image
image
image

image

</details>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+name: "Bug Report"
+description: Report a bug with Python.org website to help us improve
```
Comment (ewdurbin) [inline .github/ISSUE_TEMPLATE/BUG.yml]: I'm -1 on capitalizing URL throughout.

**Review by ewdurbin — APPROVED** [review]
only one real nit.

File: .github/ISSUE_TEMPLATE/BUG.yml (hunk 1 above)
Comment (JacobCoffee) [reply .github/ISSUE_TEMPLATE/BUG.yml]: thank goodness

Comment (JacobCoffee) [inline .github/ISSUE_TEMPLATE/BUG.yml:2]:
```suggestion
description: Report a bug with python.org website to help us improve
```

**Pull request merged**